networkx/algorithms/simple_paths.py

```diff
@@ -274,7 +275,8 @@ def all_simple_edge_paths(G, source, target, cutoff=None):
        Single node or iterable of nodes at which to end path
 
     cutoff : integer, optional
-        Depth to stop the search. Only paths of length <= cutoff are returned.
+        Depth to stop the search. Only paths containing <= ``cutoff + 1`` nodes
+        are returned.
```

Comment:
This is dealing with edge-paths... So its different. Maybe something like:
```suggestion
        Depth to stop the search. Only paths of length <= cutoff are returned.
        Note that the length of an edge path is the number of edges.
```